docs/guide/getting-started.md

```
@@ -22,9 +22,21 @@ Before you begin, make sure you have the following installed:
 - VitePress
 - OpenAPI Specification (Version 3)
 
-## Installation
+## Starter Template
+
+To get started quickly with `vitepress-openapi`, you can use the provided starter template. This template includes a basic VitePress setup with `vitepress-openapi` and a sample OpenAPI specification.
+
+You can access the template from:
+- GitHub: [https://github.com/enzonotario/vitepress-openapi-starter](https://github.com/enzonotario/vitepress-openapi-starter)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)